examples/output_proto.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import sys
+from os.path import isdir, join
+import sunbeam
+
+
+def main():
+    # es = sunbeam.parse_file(
+    #     join(opmdatadir(), 'norne/NORNE_ATW2013.DATA'),
+    #     ('PARSE_RANDOM_SLASH', sunbeam.action.ignore))
+    deck = sunbeam.parse_file(join(opmdatadir(), 'norne/NORNE_ATW2013.DATA'))
+    # for kw in deck:
+    #     print kw.name()
+    print len(deck);
+    for i in range(0,min(10, len(deck))):
+        print deck[i]
+
+
+
+
+
+
+
+
+
+################################################################################
+
+def opmdatadir():
```

Comment:
No reason to go through theese hoops with `global OPMDATA_DIR` - just take the deck input from `sys.argv`:
```python
parse( sys.argv[1] )
```